spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/AwsSecretsManagerEnvironmentProperties.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2018-2019 the original author or authors.
```

Comment:
`2020` instead `2019` and make sure all the files touched in the PR are updated

Reply:
b54e1e7